### DISCUSSION TITLE
nucleo_h745zi_q_m7 won't flash hello_world

### DISCUSSION THREAD

**Discussion by codewitch-honey-crisis** [zephyrproject-rtos/zephyr#60927] [General]
I've followed the docs here on 4 brand new nucleo_h745_q_m7 boards and get the same result on each

https://docs.zephyrproject.org/latest/boards/arm/nucleo_h745zi_q/doc/index.html

```
xPack OpenOCD x86_64 Open On-Chip Debugger 0.11.0+dev (2022-09-01-17:58)
Licensed under GNU GPL v2
For bug reports, read
        http://openocd.org/doc/doxygen/bugs.html
init
Info : STLINK V3J5M2 (API v3) VID:PID 0483:374E
Info : Target voltage: 3.304000
Error: init mode failed (unable to connect to the target)
```
This is either a bug in the documentation or a bug in the flashing system. Which is it?

**Comment by FRASTM** [comment]
I do not have that  nucleo_h745zi_q_m7  for testing
I can just suggest :
1.  to upgrade the STLINK  (with STM32CubeProgrammer)  of the nucleo board)
2.  to check there is no other connection (like STM32CubeProgrammer) to the target board
3. check the hw and sw config of the LDO/SMPS power supplies  (see below UserManual)

**Comment by FRASTM** [comment]
STLINK:

![cubeProgrammer](https://github.com/zephyrproject-rtos/zephyr/assets/51417392/8f5b41ca-649c-4336-84f4-bccc55b67401)
![stm32STLINK_update](https://github.com/zephyrproject-rtos/zephyr/assets/51417392/e2b54a91-c8bf-497e-b08f-274e02771f26)

**Comment by FRASTM** [comment]
UserManual: 
 The stm32h747i_disco board has a hw configuration that could lock the board because of power supplies when hw and sw config mismatch. You can find clarification in the UserManual [UM2488 ](https://www.st.com/content/ccc/resource/technical/document/user_manual/group1/23/54/f3/b8/10/ae/44/d9/DM00547983/files/DM00547983.pdf/jcr:content/translations/en.DM00547983.pdf ) of the disco kit.


![UM2488_stm32h747i_disco](https://github.com/zephyrproject-rtos/zephyr/assets/51417392/c8dbf488-3c90-4b82-9b30-bfee6f11bc61)

**Comment by joelguittet** [comment]
I confirm using an up to date STLink is okay for me with this board.
